content/en/blog/_posts/2021-12-15-StatefulSet-PVC-Auto-Deletion.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: blog
+title: 'Kubernetes 1.23: StatefulSet PVC Auto-Deletion (alpha)'
+date: 2021-12-15
```

Comment:
BTW did the release comms team check this date yet?

Reply:
Yeah, the release person (Mickey on slack) said I should do what https://github.com/kubernetes/website/pull/30502 did.

Reply:
Hello! K8s 1.23 Comms Lead here. We actually have this blog scheduled for 12/16 (so it comes out the day after the [honor PV reclaim policy fix blog post](https://github.com/kubernetes/website/pull/30556). Updating the publishing date. 

```suggestion
date: 2021-12-16
```

Reply:
sgtm, thx!